Makefile

```diff
@@ -71,6 +71,15 @@ for-all:
 	 	$${CMD} ); \
 	done
 
+.PHONY: $(ALL_GO_MODULES)
```

Comment:
Are these two needed or can a single definition work without making the module directories phony targets themselves?

```
.PHONY: for-all-target $(ALL_GO_MODULES)
for-all-target: $(ALL_GO_MODULES)
$(ALL_GO_MODULES):
	@echo "Running target '$(TARGET)' in module '$@'"
	$(MAKE) -C $@ $(TARGET)
```

Reply:
`.PHONY: $(ALL_GO_MODULES)` is needed otherwise the Makefile will tell you there's no need to run for all the modules as they are unchanged. By making the targets phony, you tell Makefile to not look at the folder timestamps and run the tasks no matter what.